src/main/scala/com/fulcrumgenomics/bam/TrimPrimers.scala

```diff
@@ -164,6 +186,10 @@ class TrimPrimers
     }
 
     out.close()
+    primerless_out_opt match {
+      case Some(primerless_out) => primerless_out.close()
+      case _ => ()
+    }
```

Comment:
Use a collection method (i.e. `foreach`) versus a match statement
```suggestion
    primerless_out_opt.foreach(_.close())
```